script.module.cache/addon.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.cache" name="cache" version="1.0.2" provider-name="fraser">
+    <requires>
+        <import addon="xbmc.python" version="2.26.0"/>
```

Comment:
Because of requiring at least `2.26.0` this can't be installed in Krypton, also it is available in the Krypton repo.

Reply:
Crap, missed this. @FraserChapman can you adjust and send a pull request?

Reply:
Ah sorry, yes will do that now.

Reply:
see https://github.com/xbmc/repo-scripts/pull/1124

Reply:
see: https://github.com/xbmc/repo-scripts/pull/1125